front/app/containers/Admin/projects/project/permissions/components/ModeratorList.tsx

```diff
@@ -33,14 +39,15 @@ const ModeratorList = memo(
     const authUser = useAuthUser();
 
     const handleDeleteClick =
-      (projectId: string, moderatorId: string) => (event: FormEvent) => {
+      (projectId: string, moderatorId: string) => async (event: FormEvent) => {
         event.preventDefault();
         const deleteMessage = formatMessage(
           messages.moderatorDeletionConfirmation
         );
 
         if (window.confirm(deleteMessage)) {
-          deleteProjectModerator(projectId, moderatorId);
+          await deleteProjectModerator(projectId, moderatorId);
+          queryClient.invalidateQueries({ queryKey: seatsKeys.items() });
```

Comment:
Not perfect, but it should work in most cases https://citizenlabco.slack.com/archives/CQAGMHKT3/p1678284957834189

Reply:
Maybe I'll move it to the service. Or maybe we'll think up sth else. Or maybe I'll leave it as it is now :smile:

Reply:
The service sounds better. That way, we don't track it everywhere.